lametro/templates/partials/event_day.html

```diff
@@ -0,0 +1,8 @@
+<h5 class="text-default">
+  <i class="fa fa-fw fa-calendar-o"></i> {{date | date:"D m/d/Y"}}
+  <div class="divider" />
```

Comment:
is there a difference between `<div />` and `<div></div>`?

Reply:
[TIL](https://stackoverflow.com/questions/1411182/is-there-a-difference-between-div-and-div-div) it's a difference between XHTML, HTML, and HTML5. Since not everyone is going to have it rendered in HTML5, I changed it. Thanks for catching!